src/chainstate/burn/db/sortdb.rs

```diff
@@ -738,7 +793,34 @@ const SORTITION_DB_SCHEMA_5: &'static [&'static str] = &[r#"
         amount TEXT NOT NULL,
 
         PRIMARY KEY(txid)
-    );"#];
+    );"#,
+    r#"
+    CREATE TABLE peg_out_requests (
+        txid TEXT NOT NULL,
+        vtxindex INTEGER NOT NULL,
+        block_height INTEGER NOT NULL,
+        burn_header_hash TEXT NOT NULL,
```

Comment:
Please include the extra, unparsed bytes as a `memo`  column.

Reply:
Done